keyboards/handwired/scottokeebs/scottomacrodeck/info.json

```diff
@@ -0,0 +1,52 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "ScottoMacrodeck",
+    "maintainer": "joe-scotto",
+    "development_board": "promicro",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "build" : {
+        "lto": true,
+    },
+    "matrix_pins": {
+        "direct": [
+            ["D3", "D2", "D1", null],
+            ["D0", "D4", "C6", "D7"],
+            ["E6", "B4", "B5", "F4"]
+        ]
+    },
+    "url": "https://scottokeebs.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0015",
+        "vid": "0xFEED"
```

Comment:
Vendor ID cannot be FEED for VIA support.

Reply:
Resolved by [7f4ae5e](https://github.com/qmk/qmk_firmware/pull/20946/commits/7f4ae5e66ce1e3b4e9d4865cc01145953ee9c1fa). The board doesn't need via, I'm hosting Vial on my website instead for GUI support.